tests/integration/test_sanity.py

```diff
@@ -520,10 +520,68 @@ def test_maxmemory(cluster):
     with (raises(ResponseError, match="OOM command not allowed when used memory > 'maxmemory'")):
         cluster.execute('set', 'key1', val)
 
-    cluster.execute('config', 'set', 'maxmemory', 4000000000)
+    cluster.execute('config', 'set', 'maxmemory', 0)
     cluster.execute('set', 'key1', val)
 
 
+def test_acl(cluster):
+    cluster.create(3)
+
+    cluster.execute('set', 'key', 1)
+    cluster.execute('set', 'abc', 1)
+    cluster.execute('acl', 'setuser', 'default', 'resetkeys', '(+set', '~key*)', '(+get', '~key*)')
+    cluster.execute('get', 'key')
+    with (raises(ResponseError, match="acl verification failed, can't access at least one of the keys mentioned in the command arguments")):
+        cluster.execute('get', 'abc')
+
+    cluster.execute('set', 'key', 2)
+    commit_idx = cluster.leader_node().commit_index()
+    assert cluster.node(1).raft_debug_exec('get', 'key') == b'2'
+    cluster.node(2).wait_for_commit_index(commit_idx, gt_ok=True)
```

Comment:
Maybe I'm missing something but can we replace `wait_xx` calls with a single `cluster.wait_for_unanimity()`?

Reply:
ok, done